<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>601</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="topbar">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(45, 45, 45);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 127);
border-radius: 7px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {

	background-color: rgb(45, 45, 45);
	border-radius: 5px;	
	background-position: center;	
}
QPushButton:hover {
	
	background-color: rgb(255, 102, 105);
	color: rgb(200, 200, 200);
}
QPushButton:pressed {
	background-color: rgb(170, 0, 0);
	color: rgb(200, 200, 200);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../Imagens/cil-x.png</normaloff>../Imagens/cil-x.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(45, 45, 45);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>500</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>10000</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(45, 45, 45);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>180</y>
            <width>231</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>13</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 0, 0);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>300</y>
            <width>231</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>13</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 0, 0);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>90</y>
            <width>221</width>
            <height>61</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
	border: 2px solid rgb(45, 45, 45);
	border-radius: 5px;
	padding: 15px;
	background-color: rgb(30, 30, 30);	
	color: rgb(100, 100, 100);
}
QLineEdit:hover {
	border: 2px solid rgb(55, 55, 55);}
QLineEdit:focus {
	border: 2px solid rgb(255, 207, 0);	
	color: rgb(200, 200, 200);}</string>
          </property>
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="dragEnabled">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="placeholderText">
           <string>Excluir Usuario</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>260</x>
            <y>90</y>
            <width>111</width>
            <height>61</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton {	
	background-color: rgb(50, 50, 50);
	border: 2px solid rgb(60, 60, 60);
	border-radius: 5px;
color: rgb(255, 255, 127);
}
QPushButton:hover {	
	background-color: rgb(60, 60, 60);
	border: 2px solid rgb(70, 70, 70);
}
QPushButton:pressed {	
	background-color: rgb(250, 230, 0);
	border: 2px solid rgb(250, 230, 0);	
	color: rgb(35, 35, 35);}</string>
          </property>
          <property name="text">
           <string>Excluir</string>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>30</y>
            <width>271</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>22</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 127);</string>
          </property>
          <property name="text">
           <string>Excluir Permiss√£o</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>180</y>
            <width>301</width>
            <height>271</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(67, 67, 67);

border-radius: 7px;

</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="label_9">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>180</y>
            <width>341</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>14</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(22, 22, 22);
color: rgb(255, 255, 127);
border-radius: 7px;
</string>
          </property>
          <property name="text">
           <string> Permiss√µes cadastradas</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QTableWidget" name="tableWidget">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>230</y>
            <width>261</width>
            <height>191</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>13</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>237</number>
          </attribute>
          <column>
           <property name="text">
            <string>Nome</string>
           </property>
          </column>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>421</y>
            <width>261</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>13</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 0, 0);
background-color: rgb(67, 67, 67);
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>80</y>
            <width>361</width>
            <height>81</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(67, 67, 67);

border-radius: 7px;

</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="label_11">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>141</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Usuario Logado:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_12">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>0</y>
            <width>241</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(170, 0, 0);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_3">
          <property name="geometry">
           <rect>
            <x>270</x>
            <y>460</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {	
	background-color: rgb(50, 50, 50);
	border: 2px solid rgb(60, 60, 60);
	border-radius: 5px;
color: rgb(255, 255, 127);
}
QPushButton:hover {	
	background-color: rgb(60, 60, 60);
	border: 2px solid rgb(70, 70, 70);
}
QPushButton:pressed {	
	background-color: rgb(250, 230, 0);
	border: 2px solid rgb(250, 230, 0);	
	color: rgb(35, 35, 35);}</string>
          </property>
          <property name="text">
           <string>Voltar</string>
          </property>
         </widget>
         <zorder>label</zorder>
         <zorder>label_10</zorder>
         <zorder>label_5</zorder>
         <zorder>label_6</zorder>
         <zorder>lineEdit</zorder>
         <zorder>pushButton</zorder>
         <zorder>label_8</zorder>
         <zorder>label_9</zorder>
         <zorder>tableWidget</zorder>
         <zorder>label_3</zorder>
         <zorder>label_11</zorder>
         <zorder>label_12</zorder>
         <zorder>pushButton_3</zorder>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="rodape">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 127);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>23</width>
           <height>23</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(45, 45, 45);
border-radius: 7px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>0</y>
            <width>181</width>
            <height>21</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 127);</string>
          </property>
          <property name="text">
           <string>APS - Ciencia da computa√ßao - MD5</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="file_rc.qrc"/>
 </resources>
 <connections/>
</ui>
